docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -22,53 +22,24 @@ Jobs can be launched in a few different ways:
 
 ## Relevant APIs
 
-| Name                                   | Description                                                                                                                                                     |
-| -------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| <PyObject object="define_asset_job" /> | A function for defining a job from a selection of assets.                                                                                                       |
-| <PyObject object="job" decorator />    | The decorator used to define a job.                                                                                                                             |
-| <PyObject object="JobDefinition" />    | A job definition. Jobs are the main unit of execution and monitoring in Dagster. Typically constructed using the <PyObject object="job" decorator /> decorator. |
-|                                        |                                                                                                                                                                 |
+| Name                                | Description                                                                                                                                                     |
+| ----------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <PyObject object="job" decorator /> | The decorator used to define a job.                                                                                                                             |
+| <PyObject object="JobDefinition" /> | A job definition. Jobs are the main unit of execution and monitoring in Dagster. Typically constructed using the <PyObject object="job" decorator /> decorator. |
+|                                     |                                                                                                                                                                 |
 
 ---
 
-## Creating jobs
+## Creating jobs directly from ops
 
-Jobs can be created in several ways:
+Jobs can be created in several ways. It can be built from software-defined assets or directly from ops.
```

Comment:
Again, this feels weird to me. Maybe we should discuss since it keeps coming up?